README.md

```diff
@@ -180,61 +221,111 @@ Go to AWS console -> Secrets Manager, then:
   - Click the orange "Next" button (we won't configure rotation)
   - Click the orange "Store" button to save the Secret
 
-### Upload SSL cert
+#### Upload SSL cert
 
-TODO: Does this section apply to JPL?
+> [!WARNING]
+> This step must be done by an ASF employee.
```

Comment:
I called out what steps require ASF to be in the loop, but maybe we don't need to with the top-level warning?

https://github.com/ASFHyP3/hyp3/pull/2603/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R30-R34

Reply:
Leaving as-is for now, but we may want to revisit this if developers outside ASF want to start deploying their own forks of HyP3.